plugins/modules/aci_l4l7_service_graph_template_node.py

```diff
@@ -0,0 +1,359 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2025, Tim Cragg (@timcragg)
+# Copyright: (c) 2025, Shreyas Srish (@shrsr)
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+DOCUMENTATION = r"""
+---
+module: aci_l4l7_service_graph_template_node
+short_description: Manage L4-L7 Service Graph Templates Nodes (vns:AbsNode)
+description:
+- Manage Manage L4-L7 Service Graph Templates Nodes.
+options:
+  tenant:
+    description:
+    - The name of an existing tenant.
+    type: str
+    aliases: [ tenant_name ]
+  service_graph:
+    description:
+    - The name of an existing Service Graph.
+    type: str
+  node:
+    description:
+    - The name of the Service Graph Template Node.
+    type: str
+  func_template_type:
```

Comment:
```suggestion
  functional_template_type:
```

Reply:
intended to skip the cloud specific option?

Reply:
Included cloud specific options